src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/PledgeService/Types/GetApplicationsForDownloadResponse.cs

```diff
@@ -23,10 +23,11 @@ public class Application
             public string LastName { get; set; }
             public IEnumerable<string> EmailAddresses { get; set; }
             public string BusinessWebsite { get; set; }
-            public bool IsLocationMatch { get; set; }
-            public bool IsSectorMatch { get; set; }
-            public bool IsJobRoleMatch { get; set; }
-            public bool IsLevelMatch { get; set; }
+            public bool MatchSector { get; set; }
```

Comment:
See comments in APIM layer about changing parts of the service that don't relate to your ticket. Apart from creating potential merge conflicts, the tester will not anticipate this download having changed.

Reply:
suggestions implemented with as little code changes as needed by this ticket